CHANGELOG.md

```diff
@@ -0,0 +1,127 @@
+# Git Changelog plugin changelog
```

Comment:
To keep track of changes introduced in releases.
Den 9 dec 2015 12:00 skrev "Paul Wellner Bou" <notifications@github.com>:

> In CHANGELOG.md
> <https://github.com/jenkinsci/git-changelog-plugin/pull/1#discussion_r47076750>
> :
>
> > @@ -0,0 +1,127 @@
> > +# Git Changelog plugin changelog
>
> Why do we need this changelog?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkinsci/git-changelog-plugin/pull/1/files#r47076750>
> .
>